@here/harp-geoutils/lib/coordinates/GeoBox.ts

```diff
@@ -47,8 +58,17 @@ export class GeoBox implements GeoBoxExtentLike {
      *
      * @param southWest - The south west position in geo coordinates.
      * @param northEast - The north east position in geo coordinates.
+     * @param wrapAround - If `true` wraps the coordinates around the antimeridian.
      */
-    constructor(readonly southWest: GeoCoordinates, readonly northEast: GeoCoordinates) {}
+    constructor(
+        readonly southWest: GeoCoordinates,
+        readonly northEast: GeoCoordinates,
+        wrapAround: boolean = false
```

Comment:
Do you need the user to specify this? Why not just infer it based on the coordinates, i.e. if the northEast's longitude is less than the southWest's longitude, then wrap, as Andres mentioned below.

Reply:
its default behaviour now